package.json

```diff
@@ -84,6 +84,5 @@
     "extends": [
       "@commitlint/config-conventional"
     ]
-  },
-  "packageManager": "yarn@4.2.2"
```

Comment:
Any reason to remove the package manager for this pull-request?

Reply:
Had an issue with package manager. Was removing the specified version will working and putting it back before pushing. It slipped my mind to put it back in this time. Apologies. Will Fix!